docs/source/changelog.rst

```diff
@@ -2293,12 +2495,16 @@ This is a small bugfix release due to a config change upstream.
 -  Add Client.current() (:pr:`1450`)
 -  Clean handling of restart timeouts (:pr:`1442`)
 
+.. _v1.19.1 - September 25th, 2017:
```

Comment:
Thanks @aneeshnema! This results in links that look like `...changelog.html#september-25th-2017`. Could you update this link and section title, and the occurrences below, to all use the YYYY-MM-DD format similar to what's used above?

Reply:
done